Add `server_url` parameter for self-hosting Trackio servers (#510)
gradio-app/trackio: README.md

  ```
  and keep your existing logging code.

> - **Local-first** design: dashboard runs locally by default. You can send metrics to a **Hugging Face Space** with `space_id` or to a **self-hosted Trackio server** you run yourself with `server_url` (or `TRACKIO_SERVER_URL`). If both a Space and a self-hosted URL are configured, the Space wins.

Author: ```suggestion
- **Local-first** design: dashboard runs locally by default. You can also send metrics to a Hugging Face Space with `space_id` for free or to a self-hosted Trackio server you run yourself with `server_url`
```
